src/main/java/io/jenkins/plugins/orka/client/OrkaVMConfig.java

```diff
@@ -0,0 +1,73 @@
+package io.jenkins.plugins.orka.client;
+
+import com.google.gson.annotations.SerializedName;
+
+public class OrkaVMConfig {
+
+    @SerializedName("orka_vm_name")
+    private String name;
+
+    @SerializedName("orka_cpu_core")
+    private int cpuCount;
+
+    @SerializedName("orka_base_image")
+    private String baseImage;
+
+    public OrkaVMConfig(String name, String deploymentStatus, int cpuCount, String baseImage, String image,
+            String configurationTemplate) {
```

Comment:
I cannot see to use or serialize/deserialize 3 of the construct params -> `deploymentStatus`, `image` and `configurationTemplate`. Do you need them?

Reply:
Good catch